machine_learning/logistic_regression.py

```diff
@@ -86,27 +86,27 @@ def cost_function(h: np.ndarray, y: np.ndarray) -> int:
     >>> h1 = sigmoid_function(0.3)
     >>> h2 = sigmoid_function(-4.3)
     >>> h3 = sigmoid_function(8.1)
```

Comment:
Let's make the tests two lines instead of six.
```suggestion
    >>> estimations = [sigmoid_function(0.3), sigmoid_function(-4.3), sigmoid_function(8.1)]
```